Use BerkeleyDB found in prefix, don't use system db from /usr/include
when this is newer than the one in prefix.
--- setup.py.orig	2009-09-23 16:05:51.844573298 +0200
+++ setup.py	2009-09-23 16:16:53.245234547 +0200
@@ -826,6 +826,7 @@
                             db_ver_inc_map[db_ver] = d
                             if db_setup_debug:
                                 print "db.h: found", db_ver, "in", d
+                            break
                         else:
                             # we already found a header for this library version
                             if db_setup_debug: print "db.h: ignoring", d
